src/test/java/ru/training/at/hw3/pages/HomePage.java

```diff
@@ -0,0 +1,221 @@
+package ru.training.at.hw3.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import ru.training.at.hw3.testdata.TestData;
+import ru.training.at.hw3.utils.ReaderUserDataForLogin;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+public class HomePage extends BaseClass {
+
+    public HomePage(WebDriver webDriver) {
+        this.webDriver = webDriver;
+    }
+
+    @FindBy(id = "user-icon")
+    private WebElement loginButton;
+
+    @FindBy(id = "name")
+    private WebElement loginField;
+
+    @FindBy(id = "password")
+    private WebElement passwordField;
+
+    @FindBy(xpath = "//*[@id='login-button']")
+    private WebElement enterButton;
+
+    @FindBy(css = "span[class='icons-benefit icon-practise']")
+    private WebElement imageFirstHomePage;
+
+    @FindBy(css = "span[class='icons-benefit icon-custom']")
+    private WebElement imageSecondHomePage;
+
+    @FindBy(css = "span[class='icons-benefit icon-multi']")
+    private WebElement imageThirdHomePage;
+
+    @FindBy(css = "span[class='icons-benefit icon-base']")
+    private WebElement imageFourthHomePage;
```

Comment:
Why do you do not use here `List<WebElement>`?

```java
@FindBy(css = "span.icons-benefit")
List<WebElement> images;

public List<WebElement> getImages() {
    return images;
}
```

In the tests it could be used as 
```java

for (WebElement image : homePage) {
    softAssert.assertTrue(image.isDisplayed());
}
```

Instead of using following elements